Index: carDecorator/src/edu/sehir/Main.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/carDecorator/src/edu/sehir/Main.java b/carDecorator/src/edu/sehir/oop/CarDecoratorMain.java
rename from carDecorator/src/edu/sehir/Main.java
rename to carDecorator/src/edu/sehir/oop/CarDecoratorMain.java
--- a/carDecorator/src/edu/sehir/Main.java	
+++ b/carDecorator/src/edu/sehir/oop/CarDecoratorMain.java	
@@ -1,6 +1,6 @@
-package edu.sehir;
+package edu.sehir.oop;
 
-public class Main {
+public class CarDecoratorMain {
 
     public static void main(String[] args) {
 
Index: carDecorator/src/edu/sehir/BasicCar.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/carDecorator/src/edu/sehir/BasicCar.java b/carDecorator/src/edu/sehir/oop/BasicCar.java
rename from carDecorator/src/edu/sehir/BasicCar.java
rename to carDecorator/src/edu/sehir/oop/BasicCar.java
--- a/carDecorator/src/edu/sehir/BasicCar.java	
+++ b/carDecorator/src/edu/sehir/oop/BasicCar.java	
@@ -1,4 +1,4 @@
-package edu.sehir;
+package edu.sehir.oop;
 
 /**
  * Created by ensar on 08.12.2016.
Index: carDecorator/src/edu/sehir/LuxuryCar.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/carDecorator/src/edu/sehir/LuxuryCar.java b/carDecorator/src/edu/sehir/oop/LuxuryCar.java
rename from carDecorator/src/edu/sehir/LuxuryCar.java
rename to carDecorator/src/edu/sehir/oop/LuxuryCar.java
--- a/carDecorator/src/edu/sehir/LuxuryCar.java	
+++ b/carDecorator/src/edu/sehir/oop/LuxuryCar.java	
@@ -1,4 +1,4 @@
-package edu.sehir;
+package edu.sehir.oop;
 
 /**
  * Created by ensar on 08.12.2016.
Index: carDecorator/src/edu/sehir/Car.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/carDecorator/src/edu/sehir/Car.java b/carDecorator/src/edu/sehir/oop/Car.java
rename from carDecorator/src/edu/sehir/Car.java
rename to carDecorator/src/edu/sehir/oop/Car.java
--- a/carDecorator/src/edu/sehir/Car.java	
+++ b/carDecorator/src/edu/sehir/oop/Car.java	
@@ -1,4 +1,4 @@
-package edu.sehir;
+package edu.sehir.oop;
 
 /**
  * Created by ensar on 08.12.2016.
Index: carDecorator/carDecorator.iml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/carDecorator/carDecorator.iml b/carDecorator/carDecorator.iml
new file mode 100644
--- /dev/null	
+++ b/carDecorator/carDecorator.iml	
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="JAVA_MODULE" version="4">
+  <component name="NewModuleRootManager" inherit-compiler-output="true">
+    <exclude-output />
+    <content url="file://$MODULE_DIR$">
+      <sourceFolder url="file://$MODULE_DIR$/src" isTestSource="false" />
+    </content>
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+  </component>
+</module>
\ No newline at end of file
Index: carDecorator/src/edu/sehir/SportsCar.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/carDecorator/src/edu/sehir/SportsCar.java b/carDecorator/src/edu/sehir/oop/SportsCar.java
rename from carDecorator/src/edu/sehir/SportsCar.java
rename to carDecorator/src/edu/sehir/oop/SportsCar.java
--- a/carDecorator/src/edu/sehir/SportsCar.java	
+++ b/carDecorator/src/edu/sehir/oop/SportsCar.java	
@@ -1,4 +1,4 @@
-package edu.sehir;
+package edu.sehir.oop;
 
 /**
  * Created by ensar on 08.12.2016.
Index: carDecorator/src/edu/sehir/CarDecorator.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/carDecorator/src/edu/sehir/CarDecorator.java b/carDecorator/src/edu/sehir/oop/CarDecorator.java
rename from carDecorator/src/edu/sehir/CarDecorator.java
rename to carDecorator/src/edu/sehir/oop/CarDecorator.java
--- a/carDecorator/src/edu/sehir/CarDecorator.java	
+++ b/carDecorator/src/edu/sehir/oop/CarDecorator.java	
@@ -1,4 +1,4 @@
-package edu.sehir;
+package edu.sehir.oop;
 
 /**
  * Created by ensar on 08.12.2016.
